cartographer/sensor/ordered_multi_queue.h

```diff
@@ -50,15 +50,20 @@ inline std::ostream& operator<<(std::ostream& out, const QueueKey& key) {
 }
 
 // Maintains multiple queues of sorted sensor data and dispatches it in merge
-// sorted order. This class is thread-compatible.
+// sorted order. It will wait to see at least one value for each unfinished queue before
+// dispatching the next time ordered value across all queues. This class is thread-compatible.
 class OrderedMultiQueue {
  public:
   using Callback = std::function<void(std::unique_ptr<Data>)>;
 
-  // Will wait to see at least one value for each unfinished queue before
-  // dispatching the next smallest value across all queues.
   OrderedMultiQueue() {}
-  ~OrderedMultiQueue() {}
+
+  // CHECKs that all queues are finished.
```

Comment:
Should this comment be there? The code is saying this clearly, which is preferable.